### DISCUSSION TITLE
It should be better to write `npm i @duckdb/duckdb-wasm@next` to the doc because @duckdb/duckdb-wasm@latest is too old and not compatible with duckdb@latest

### DISCUSSION THREAD

**Discussion by Narazaka** [duckdb/duckdb-wasm#1878] [General]
Because I fell into the trap and wrote bitter words on SNS.

**Comment by domoritz** [comment]
I think instead a new release would be better.

**Reply by carlopi** [reply]
New release has to be.
I was doing some more serious testing, one problem I am aware of in current main is that multithreaded is at the moment broken, due to API changes Emscripten side that we need to adapt to. This affects only `coi` target, unsure if that's a blocker or given situation release is still better.

**Reply by carlopi** [reply]
I am trying to fix that, then another round of testing and marking a release.

**Comment by mbostock** [comment]
Related https://github.com/duckdb/duckdb-wasm/issues/1994
